Objective-C/CBLReplicator.mm

```diff
@@ -471,14 +479,26 @@ - (void) setProgressLevel: (CBLReplicatorProgressLevel)level {
 - (BOOL) isDocumentPending: (NSString *)documentID
                 collection: (CBLCollection *)collection
                      error: (NSError**)error {
+    CBLAssertNotNil(collection);
     
-    // TODO: Add implementation 
+    CBLStringBytes name(collection.name);
+    CBLStringBytes scopeName(collection.scope.name);
+    C4CollectionSpec c4spec = { .name = name, .scope = scopeName };
     
-    return NO;
+    return [self isDocumentPending: documentID c4Spec: c4spec error: error];
 }
 
 - (BOOL) isDocumentPending: (NSString*)documentID error: (NSError**)error {
     CBLAssertNotNil(documentID);
+    
+    return [self isDocumentPending: documentID c4Spec: kC4DefaultCollectionSpec error: error];
+}
+
+// internal
+- (BOOL) isDocumentPending:(NSString *)documentID
```

Comment:
Why doesn't this get implemented in `- (BOOL) isDocumentPending: (NSString*)documentID collection: error: (NSError**)error` itself?